src/nrniv/nrnmenu.cpp

```diff
@@ -1243,6 +1250,10 @@ bool MechanismType::is_artificial(int i) {
     return (nrn_is_artificial_[j] ? true : false);
 }
 
+bool MechanismType::is_ion() {
+    return nrn_is_ion(internal_type());
```

Comment:
@nrnhines : I hope this is correct i.e. `is_ion()` doesn't require any argument like `is_artificial()` because all instance types are either IONs or not.